zebra-chain/src/transaction.rs

```diff
@@ -146,7 +147,7 @@ impl Transaction {
         network_upgrade: NetworkUpgrade,
         hash_type: sighash::HashType,
         input: Option<(u32, transparent::Output)>,
-    ) -> blake2b_simd::Hash {
+    ) -> SignatureHash {
```

Comment:
This is short and clear what kind of Hash we're talking about

```suggestion
    ) -> sighash::Hash {
```

Reply:
Done in 463f5f29